gatsby-config.js

```diff
@@ -14,23 +14,19 @@ module.exports = {
     'gatsby-plugin-postcss',
     'gatsby-plugin-image',
     'gatsby-plugin-sharp',
+    'gatsby-transformer-json',
     {
-      resolve: 'gatsby-transformer-yaml',
+      resolve: 'gatsby-source-filesystem',
       options: {
-        typeName: ({ node }) => {
-          const name = node.sourceInstanceName;
-          if (name === 'radschnellwege') {
-            return 'Radschnellweg';
-          }
-          return name;
-        },
+        path: './src/radschnellwege/metadata',
```

Comment:
FYI in https://github.com/FixMyBerlin/fixmy.fahrradparken/blob/develop/gatsby-config.js#L61 habe ich das kürzlich überall auf das `__direname` pattern gewechselt. Es scheint keinen Unterschied zu machen, aber alle Gatsby docs verwenden es.